5.5/root/usr/bin/run-mysqld

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+export_vars=$(cgroup-limits); export $export_vars
```

Comment:
I don't know, I've used the same code as @mfojtik. My idea was to do the same things similar in a different places.